README.md

```diff
@@ -1,2 +1,30 @@
 # comp110-worksheet-B
-Base repository for COMP110 worksheet B
+
+Fallout 4 hacking mechanic
+
+word = guess
+cword = correct word
+
+Enter terminal
+input attempt
+
+if word contains 0 likeness
+	then display 0 Likeness and - 1 number attempts
+		else if num.attempts > 5 then lock terminal for 10s and reset word
+
+if word contains > 1 letters similar to cword then
+	display "likeness 1"
+	
+if word > contains > 2 letters similar to cword then
+	display "likeness 2"
+	
+if word > contains > 3 letters similar to cword then
+	display "likeness 3"
+	
+if word > contains > 4 letters similar to cword then
+	display "likeness 4"
+	
+if word > contains > 5 letters similar to cword then
```

Comment:
This assumes the password always has exactly 5 letters. This type of approach would be difficult to scale to different numbers of letters. Code structures such as loops would help here.